public/js/ticTacToe.js

```diff
@@ -3,6 +3,61 @@ const playerTurn = 'X'
 const Xmoves = []
 const Omoves = []
 
-const boardGame = document.querySelector('.board')
+const boardGame = document.querySelector('#board')
 const resetButton = document.querySelector('#reset')
+const divBoxes = document.querySelectorAll('div.box')
+const winningCombos = [[1,2,3], [4,5,6], [7,8,9], [1,4,7], [1,5,9], [3,5,7], [3,6,9], [2,5,8]]
 
+const switchTurn = function() {
+  if(playerTurn === 'X') {
+    playerTurn = 'O'
+  } else {
+    playerTurn = 'X'
+  }
+}
+
+for(let i = 0; i < divBoxes.length; i++) {
+  divBoxes[0].getAttribute("id")
+}
+
+const playerMoves = function() {
+  if(playerTurn === 'X') {
+    console.log("it's your turn", playerTurn)
+    this.innerText = "X"
+    Xmoves.push(parseInt(this.getAttribute("id")))
+    switchTurn()
+    console.log("Xmoves", Xmoves)
+    checkWinning(Xmoves, "X Player")
+    counter++
+    console.log("counter", counter)
+    checkCounter()
+  } else {
+    console.log("it's your turn", playerTurn)
+    this.innerText = "O"
+    Omoves.push(parseInt(this.getAttribute("id")))
+    switchTurn()
+    console.log("Omoves", Omoves)
+    checkWinning(Omoves, "Omoves")
+    counter++
+    console.log("counter", counter)
+    checkCounter()
+  }
+}
```

Comment:
Notice that this `playerMoves` function has a lot of repeating code. It's only repeated once, so it's going to be a matter of preference / opinion, you may choose to abstract it one level, or you may choose to leave it as is. If it ever grows to beyond just two, it's good to know how to abstract functions.

I'll give you an example of playerMoves that's abstracted for either player, just so you know what it 'could' look like.

```
const moves = {
  X: [],
  O: []
}

// ... You're other code

const playerMoves = function() {
  console.log("it's your turn", playerTurn)
  this.innerText = playerTurn 
  moves[playerTurn].push(parseInt(this.getAttribute("id")))
  switchTurn()
  console.log(playerTurn+'moves', moves[playerTurn])
  checkWinning(moves[playerTurn], playerTurn+" Player")
  counter++
  console.log("counter", counter)
  checkCounter()
 }
```